data/mods/Aftershock/mobs/aliens.json

```
@@ -151,5 +151,41 @@
     "anger_triggers": [ "STALK", "FRIEND_ATTACKED", "FRIEND_DIED", "PLAYER_WEAK", "PLAYER_CLOSE" ],
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE" ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "afs_titanis",
+    "name": { "str": "titanis" },
+    "species": "BIRD",
+    "default_faction": "bear",
+    "bodytype": "ostrich",
+    "symbol": "T",
+    "color": "blue_green",
+    "volume": "92500 ml",
+    "weight": "120000 g",
+    "material": "flesh",
+    "aggression": -10,
+    "morale": 60,
+    "speed": 150,
+    "melee_skill": 8,
+    "melee_dice": 1,
+    "melee_dice_sides": 8,
+    "melee_cut": 2,
+    "dodge": 2,
+    "armor_bash": 1,
+    "armor_cut": 1,
+    "luminance": 0,
+    "hp": 60,
+    "special_freq": [ 0 ],
+    "death_function": "NORMAL",
+    "special_attacks": [ { "type": "leap", "cooldown": 5, "max_range": 5, "allow_no_target": true }, { "type": "bite", "cooldown": 10 } ],
+    "description": "A tall bird with two long, muscular legs, a long neck, and a long sharp beak.",
+    "flags": [ "SEES", "SMELLS", "HEARS", "ANIMAL", "PATH_AVOID_DANGER_1", "GRABS", "KEENNOSE", "WARM", "HIT_AND_RUN" ],
+    "biosignature": { "biosig_item": "feces_bird", "biosig_timer": 3 },
+    "harvest": "bird_large",
+    "anger_triggers": [ "PLAYER_WEAK", "HURT" ],
+    "fear_triggers": [ "SOUND", "FIRE" ],
+    "placate_triggers": [ "MEAT" ],
```

This is assuming that these are not particularly bad-tempered but not particularly cowardly either.
```suggestion
    "anger_triggers": [ "PLAYER_WEAK", "HURT", "FRIEND_ATTACKED", "PLAYER_CLOSE" ],
    "fear_triggers": [ "FIRE", "FRIEND_DIED" ],
```
Anger triggers: With an anger of -10, it will usually require at least 2 of the above triggers going off for them to attack, with the exception of against things their faction hates - small animals and bees. "HURT" or "PLAYER_CLOSE" could be dropped from the above to make them less aggressive, although note with HIT_AND_RUN that after any attack they'll be fleeing for a bit in any case.
Fear triggers: SOUND is in the fear triggers for the BIRD species (see species.json), as are HURT and FRIEND_ATTACKED; with a morale of 60, it would usually require at least 4 of these going off for them to flee _just_ due to it - they will also flee if anger is <= 0 and they're below full HP... or after a melee hit given HIT_AND_RUN.
Placate triggers: MEAT doesn't work in CDDA currently, although it won't give an error either.